Mathlib/RingTheory/RootsOfUnity/EnoughRootsOfUnity.lean

```diff
@@ -75,6 +75,17 @@ lemma natCard_rootsOfUnity (M : Type*) [CommMonoid M] (n : ℕ) [NeZero n]
     simp only [mem_rootsOfUnity, PNat.mk_coe]
     rw [← Units.eq_iff, Units.val_pow_eq_pow_val, IsUnit.unit_spec, h.pow_eq_one, Units.val_one]
 
+lemma map_of_rootsOfUnityEquiv {n : ℕ} [NeZero n] {M N : Type*} [CommMonoid M] [CommMonoid N]
+    (hm : HasEnoughRootsOfUnity M n) (e : rootsOfUnity n M ≃* rootsOfUnity n N) :
```

Comment:
Please consider this suggested name, to enable dot-notation. (Maybe move it outside the namespace, so that you can remove the `_root_.`)
```suggestion
lemma _root_.MulEquiv.hasEnoughRootsOfUnity {n : ℕ} [NeZero n] {M N : Type*} [CommMonoid M] [CommMonoid N]
    [hm : HasEnoughRootsOfUnity M n] (e : rootsOfUnity n M ≃* rootsOfUnity n N) :
```
I think you can take `hm` as instance implicit argument.